[docs] Formatting and proper noun tune-up (#4200)
drud/ddev: docs/content/users/basics/troubleshooting.md

# Troubleshooting

Things might go wrong! In addition to this page, consider checking [Stack Overflow](https://stackoverflow.com/tags/ddev) and [the DDEV issue queue](https://github.com/drud/ddev/issues) and [other support options](../support.md), as well as [Docker troubleshooting suggestions](../install/docker-installation.md#testing-and-troubleshooting-your-docker-installation).

## General Troubleshooting Strategies

* Start with a `ddev poweroff` to make sure all containers can start fresh.
* Temporarily disable firewalls, VPNs, network proxies, and virus checkers while you’re troubleshooting.
* Temporarily disable any proxies you’ve established in Docker’s settings.
* Use `ddev debug dockercheck` and `ddev debug test` to help sort out Docker problems.

> * On macOS and traditional Windows, check to make sure Docker Desktop is not out of disk space. In *Settings* (or *Preferences*) → *Resources* → *Disk image size* there should be ample space left; try not to let usage exceed 80% because the reported number can be unreliable. If it says zero used, something is wrong.

Reviewer: ```suggestion
* On macOS, check to make sure Docker Desktop or Colima are not out of disk space. In *Settings* (or *Preferences*) → *Resources* → *Disk image size* there should be ample space left; try not to let usage exceed 80% because the reported number can be unreliable. If it says zero used, something is wrong.
```

* Missing `index.php`: There may not be an `index.php` or `index.html` in your project.

## `ddev start` Fails and Logs Contain "failed (28: No space left on device)" - Docker File Space

If `ddev start` fails, it’s most often because the `web` or `db` container fails to start. In this case, the error message from `ddev start` says something like “Failed to start <project>: db container failed: log=, err=container exited, please use 'ddev logs -s db' to find out why it failed”. You can`ddev logs -s db` to find out what happened.

If you see any variant of “no space left on device” in the logs when using Docker Desktop, it means you have to increase or clean up Docker’s file space. Increase the “Disk image size” setting under “Resources” in Docker’s Preferences: ![Docker disk space](../images/docker_disk_space.png).

If you see “no space left on device” on Linux, it most likely means your filesystem is full.

## `ddev start` Fails with "container failed to become ready"

A container fails to become ready when its health check is failing. This can happen to any of the containers, and you can usually find the issue with a `docker inspect` command.

> !!!tip

Reviewer: I didn't understand why you sometimes use a bare "tip" and other times put "what the tip is about" after it.

Author: Efficiency. If I could convey the content of the tip in a short title or phrase, I’d use the tip heading and otherwise leave it bare. Two reasons:

1. **Balance.** A *long*, bold heading is more disruptive to the page flow. Tips already get visual emphasis, so a wordy bold heading pushes the whole thing too far away from *helpful aside* and into the realm of *visual assault*.
2. **Skimming.** The heading can help me know whether to read or skip the supporting text, which is vital if I’m scanning for information I need.
